stl/inc/limits

```diff
@@ -1058,9 +1058,14 @@ _NODISCARD int _Checked_x86_x64_countr_zero(const _Ty _Val) noexcept {
     constexpr _Ty _Max    = (numeric_limits<_Ty>::max)();
 
 #ifndef __AVX2__
-    const bool _Definitely_have_tzcnt = __isa_available >= __ISA_AVAILABLE_AVX2;
-    if (!_Definitely_have_tzcnt && _Val == 0) {
-        return _Digits;
+    // Because the widening done below will always give a non-0 value, checking for tzcnt
+    // is not required on 16-bits and smaller since the only difference in behavior between
+    // bsf and tzcnt is when the value is 0.
+    if _CONSTEXPR_IF (_Digits > 16) {

```

Comment:
Let's let this PR wait until after #1544 and change this `_CONSTEXPR_IF` to just `constexpr`.

Reply:
Fair enough, I was watching that one too to fix the merge conflict that would arise.

Reply:
Updated with just constexpr 👍